src/components/display/fewlines/MobileNavigationBar/MobileNavigationBar.tsx

```diff
@@ -53,7 +53,11 @@ export const MobileNavigationBar: React.FC<MobileNavigationBarProp> = ({
             </ListItemLabel>
             <RightChevron />
           </ListItem>
-          <ListItem>
+          <ListItem
+            onClick={() =>
+              router.push("/account/security").then(() => setIsOpen(false))
```

Comment:
Not right now but put a pin in that: I fear this render all links useless until react is loaded, which is kind of a shame